gufe/vendor/pdb_file/pdbfile.py

```diff
@@ -40,6 +40,7 @@
 from datetime import date
 
 import numpy as np
+from openmm import Platform
```

Comment:
It bafflles me as to why we vendor from OpenMM and then also import from it in the vendor 🤦🏽 (this is a complaint against the concept of this file, not this PR).

Again, something we're better off dealing with later.